data/js/server/house/house.js

```diff
@@ -74,6 +88,14 @@ function onHouseCommand( pSocket, iMulti, cmdID )
 /** @type { ( left: Multi, leaving: BaseObject ) => boolean } */
 function onEntrance( iMulti, charEntering, objType )
 {
+	//Start Decay timer if decay is enabled or init is not true.
+	if( !iMulti.GetTag( "init" ))
```

Comment:
"init" is too generic name for a tag. Other scripts could also be on houses that require initialization, and would need to do their own checking vs such a tag. Make this more unique to the feature, like... "houseDecayInit" for example.